### PR TITLE
Adding path to avm

### PR DESCRIPTION
**Pull request by svashisht03**

### CODE REVIEW

**Review by shoekstra — DISMISSED** [review]

File: main.tf [hunk 1]
```diff
@@ -106,6 +107,7 @@ module "additional_tfe_workspaces" {
   global_remote_state            = each.value.global_remote_state
   name                           = coalesce(each.value.name, each.key)
   oauth_token_id                 = coalesce(each.value.vcs_oauth_token_id, var.tfe_workspace.vcs_oauth_token_id)
+  path                           = coalesce(each.value.path)
```
Comment (marwinbaumannsbp) [inline main.tf:110]:
```suggestion
  path                           = coalesce(each.value.path, var.tfe_workspace.path)
```

Comment (marwinbaumannsbp) [reply main.tf:110]: This allows for only setting the path once and enable this for both workspaces if needed

File: CHANGELOG.md [hunk 2]
```diff
@@ -4,6 +4,10 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](http://keepachangelog.com/) and this project adheres to [Semantic Versioning](http://semver.org/).
 
+## 2.3.2 (2021-01-18)
```
Comment (marwinbaumannsbp) [inline CHANGELOG.md:7]: https://github.com/schubergphilis/terraform-aws-mcaf-avm/pull/36 got merged a few hours ago, can you rebase and version this 2.5.0?

**Review by marwinbaumannsbp — CHANGES_REQUESTED** [review]

File: main.tf (hunk 1 above)
Comment (svashisht03) [reply main.tf:111]: Thanks for pointing out. I missed that.

File: CHANGELOG.md (hunk 2 above)
Comment (svashisht03) [reply CHANGELOG.md:12]: Done

**Review by shoekstra — APPROVED** [review]

**Pull request merged**